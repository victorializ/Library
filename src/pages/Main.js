import React from 'react';
import { Header } from 'semantic-ui-react'

import { constants } from '../i18n';

import image from "../assets/images/books.jpg";

function Main() {
  return (
    <>
      <img 
        src={image} 
        alt="Library"
        className="main__image">  
      </img>
      <Header 
        as='h1' 
        textAlign="center"
        className="main__label">
        {constants.title}
      </Header>
    </>
  );
}

export { Main };