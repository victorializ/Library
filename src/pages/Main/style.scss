.main {
    &__image {
        height: 85%;
        width: 100%;
    }
}