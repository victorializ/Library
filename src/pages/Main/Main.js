import React, { Fragment } from 'react';
import { Header } from 'semantic-ui-react'

import { constants  } from '../../i18n';

import image from "../../assets/images/books.jpg";
import './style.scss';

function Main() {
  return (
    <Fragment>
        <img src={image} alt="Library" class="main__image"></img>
        <Header as='h1' textAlign='center'>{constants.title}</Header>
    </Fragment>
  );
}

export { Main };