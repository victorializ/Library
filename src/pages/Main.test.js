import React from 'react';
import { shallow } from 'enzyme';

import { Main } from './Main';

describe('Main page component', () => {
    it('renders correctly', () => {
        expect(shallow(<Main />)).toMatchSnapshot();
    }); 
});