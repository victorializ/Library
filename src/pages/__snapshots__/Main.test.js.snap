// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page component renders correctly 1`] = `
<div
  className="main"
>
  <img
    alt="Library"
    className="main__image"
    src="books.jpg"
  />
  <Header
    as="h1"
    className="main__label"
    textAlign="center"
  >
    Library
  </Header>
</div>
`;
